name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/AccountContextMenu.java

```diff
@@ -49,13 +49,6 @@ public void menuAboutToShow(IMenuManager manager, final Account account, final S
 
         manager.add(new Separator());
 
-        new OpenDialogAction(owner, Messages.AccountMenuTransfer) //
```

Comment:
This change - the deletion of the the account transfer - I do not understand. I do not see it moved to another location.